src/txn/txn_timestamp.c

```diff
@@ -302,6 +302,32 @@ __txn_global_query_timestamp(
 done:	__wt_timestamp_set(tsp, &ts);
 	return (0);
 }
+
+/*
+ * __txn_query_timestamp --
+ *	Query a timestamp.
```

Comment:
Here (and the next function) I'd like the comment to clearly state "Query a timestamp within this session's txn." so that it isn't confused with the connection/global function.